### PR TITLE
fix: Remove any one of the two progress bars #705

### LINKED ISSUE #705: Remove any one of the two progress bars
**Is it a ?**

- [x] Bug Report

- [ ] Feature Request
    
- [ ] Chore
    
### Bug Report    

**Actual Behaviour**

When swiped down to refresh, two progress bars appear simultaneously (one from the SwipeRefreshLayout, and other from the "included" progressbar_layout.xml)

**Expected Behaviour**

Only one of the progress bars should be displayed as double progress bars make the UI look odd.

**Steps to reproduce it**

Swipe down to refresh

**Screenshot of the issue**

doublerefresh

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mishuvs**
Fixes #705 

Changes: Disable the SwipeRefreshLayout's ProgressBar once refresh is activated so that only the bigger ProgressBar (center) is visible while refreshing.

Screenshots for the change: 
refresh

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Can you add gif of the attendees page, showing how if looks in case there are no items?

File: app/src/main/java/org/fossasia/openevent/app/module/event/dashboard/EventDashboardFragment.java [hunk 1]
```diff
@@ -135,8 +135,10 @@ private void setupRefreshListener() {
         container = binding.container;
         refreshLayout = binding.swipeContainer;
         refreshLayout.setColorSchemeColors(utilModel.getResourceColor(R.color.color_accent));
-        refreshLayout.setOnRefreshListener(() ->
-            getPresenter().loadDetails(true)
+        refreshLayout.setOnRefreshListener(() -> {
+            refreshLayout.setRefreshing(false);
+            getPresenter().loadDetails(true);
+            }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/event/dashboard/EventDashboardFragment.java]: Move parenthesis below to this line

File: app/src/main/java/org/fossasia/openevent/app/module/event/dashboard/EventDashboardFragment.java [hunk 2]
```diff
@@ -135,9 +135,10 @@ private void setupRefreshListener() {
         container = binding.container;
         refreshLayout = binding.swipeContainer;
         refreshLayout.setColorSchemeColors(utilModel.getResourceColor(R.color.color_accent));
-        refreshLayout.setOnRefreshListener(() ->
-            getPresenter().loadDetails(true)
-        );
+        refreshLayout.setOnRefreshListener(() -> {
+            refreshLayout.setRefreshing(false);
+            getPresenter().loadDetails(true);
+            });
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/event/dashboard/EventDashboardFragment.java]: Indentation should be 1 less

**Comment by mishuvs** [comment]
Yes! And fixed indentation @iamareebjamal 
attendies

**Pull request merged**